src/System.Management.Automation/engine/NativeCommandParameterBinder.cs

```diff
@@ -198,16 +198,16 @@ internal bool UseArgumentList
                         // This will default to the new behavior if it is set to anything other than Legacy
                         var preference = LanguagePrimitives.ConvertTo<NativeArgumentPassingStyle>(
                             Context.GetVariableValue(new VariablePath(SpecialVariables.NativeArgumentPassing), NativeArgumentPassingStyle.Standard));
```

Comment:
This variable path should be a static readonly so we don't need to allocate a new one every time